From: Hilko Bengen <bengen@debian.org>
Date: Fri, 25 Apr 2014 09:40:53 +0200
Subject: Disable pcap compatibility check

---
 configure.ac       | 16 ----------------
 nping/configure.ac | 16 ----------------
 2 files changed, 32 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0852e24..daf28ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -395,22 +395,6 @@ if test $needs_cpp_precomp = yes; then
   CXXFLAGS="-no-cpp-precomp $CXXFLAGS"
 fi
 
-if test $have_libpcap = yes; then
-  if test "${LIBPCAP_INC+set}" = "set"; then
-    _cflags=$CXXFLAGS
-    _ldflags=$LDFLAGS
-
-    CPPFLAGS="-I$LIBPCAP_INC $CPPFLAGS"
-    LDFLAGS="-L$LIBPCAP_LIB $LDFLAGS"
-  fi
-
-  # link with -lpcap for the purposes of this test
-  LIBS_OLD="$LIBS"
-  LIBS="$LIBS -lpcap"
-  PCAP_IS_SUITABLE([have_libpcap=yes], [have_libpcap=no], [have_libpcap=yes])
-  LIBS="$LIBS_OLD"
-fi
-
 if test $have_libpcap != yes; then
   AC_CONFIG_SUBDIRS(libpcap)
   if test "${LIBPCAP_INC+set}" = "set"; then
diff --git a/nping/configure.ac b/nping/configure.ac
index b6347b1..e020a31 100644
--- a/nping/configure.ac
+++ b/nping/configure.ac
@@ -225,22 +225,6 @@ if test $needs_cpp_precomp = yes; then
   CXXFLAGS="-no-cpp-precomp $CXXFLAGS"
 fi
 
-if test $have_libpcap = yes; then
-  if test "${LIBPCAP_INC+set}" = "set"; then
-    _cflags=$CXXFLAGS
-    _ldflags=$LDFLAGS
-
-    CPPFLAGS="-I$LIBPCAP_INC $CPPFLAGS"
-    LDFLAGS="-L$LIBPCAP_LIB $LDFLAGS"
-  fi
-
-  # link with -lpcap for the purposes of this test
-  LIBS_OLD="$LIBS"
-  LIBS="$LIBS -lpcap"
-  PCAP_IS_SUITABLE([have_libpcap=yes], [have_libpcap=no], [have_libpcap=yes])
-  LIBS="$LIBS_OLD"
-fi
-
 if test $have_libpcap != yes; then
   if test "${LIBPCAP_INC+set}" = "set"; then
     LDFLAGS="-L$libpcapdir $_ldflags"
